db_specified_launcher.py

```diff
@@ -25,37 +24,22 @@
 import time
 import datetime
 
-# PSRDB imports
-from tables import *
-from joins import *
 from graphql_client import GraphQLClient
+
```

Comment:
Is it neccessary to explicitly list all the individual imports? The * import commands are used throughout the pipeline code and seem adequate, and I'm not sure why you chose this spot to itemise them. Is there an advantage to doing this?

Reply:
`import *` is bad practice because it is hard to trace back where you are importing your functions from. For example, I was unsure where most of these functions were from and it took me a while to work out they were PSRDB functions. The worst-case scenario is that you have two libraries that have a function with the same name, and you import them both, but it is hard to tell which is which. 

I will likely remove all the `import *` at some point to correct this.